sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVOptions.scala

```diff
@@ -26,7 +26,7 @@ import org.apache.commons.lang3.time.FastDateFormat
 import org.apache.spark.internal.Logging
 import org.apache.spark.sql.catalyst.util.{CaseInsensitiveMap, CompressionCodecs, ParseModes}
 
-private[csv] class CSVOptions(
+private[sql] class CSVOptions(
```

Comment:
`execution` package itself is meant to be a private according to https://github.com/apache/spark/commit/511f52f8423e151b0d0133baf040d34a0af3d422 if this should be accessible other than `csv` package.